bundles/model/org.eclipse.smarthome.model.script/META-INF/MANIFEST.MF

```diff
@@ -19,6 +19,7 @@ Require-Bundle: org.eclipse.xtext;visibility:=reexport,
  org.eclipse.smarthome.model.lazygen;resolution:=optional
 Import-Package: com.google.common.base,
  com.google.common.collect,
+ javax.annotation,
```

Comment:
You should mark it optional, so that it is only required at compile time, but not at runtime.